### ISSUE TITLE
Cannot open flika on macOS Catalina

### ISSUE THREAD

**Issue by csamuel11** [flika-org/flika#49]
I am using a mac with macOS Catalina. I am using anaconda to access python. In the terminal, I typed "pip install flika" as directed. Then, I typed "ipython". Finally, I typed "from flika import *", then "start_flika()". Up to this point everything is okay. However, after clicking return, I get a long string a text that ends with the following: KeyError: 'Unknown configuration option "useWeave"'. Could you please help? I attached the code in a Word document.
[error.docx](https://github.com/flika-org/flika/files/6233174/error.docx)

**Comment by WeiyuChen-AhrensLab** [comment]
Hi there,
I'm encountering a similar problem as csamuel11: when I launch flika from a notebook I get the error posted below which ends with:
`KeyError: 'Unknown configuration option "useWeave"'`
```
~/anaconda3/lib/python3.9/site-packages/flika/window.py in <module>
     11 from .utils.BaseProcess import WindowSelector, SliderLabel
     12 
---> 13 pg.setConfigOptions(useWeave=False)
```

Could this be because of version incompatibilities?

I have the following versions installed:

flika                  0.2.30
pyqt                      5.9.2            py37h05f1152_2  
pyqt5                     5.15.4                   pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.8.1                   pypi_0    pypi
pyqtgraph                 0.12.1                   pypi_0    pypi

Really appreciate any support on this (this package is really helpful for my work).

Thanks,

Weiyu

-------
This is my full error trace - let me know if I can provide any more information to fix the issue.

```
Starting flika
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-1-a5cf653f9f79> in <module>
     13 reload(zds)
     14 reload(fs)
---> 15 flk.start_flika()

~/anaconda3/lib/python3.9/site-packages/flika/flika.py in start_flika(files)
    113     logger.debug("Started 'flika.start_flika()'")
    114     print('Starting flika')
--> 115     fa = FlikaApplication()
    116     load_files(files)
    117     fa.start()

~/anaconda3/lib/python3.9/site-packages/flika/app/application.py in __init__(self)
    151     def __init__(self):
    152         logger.debug("Started 'creating app.application.FlikaApplication'")
--> 153         from ..process.file_ import open_file, open_file_from_gui, open_image_sequence_from_gui, open_points, save_file, save_movie_gui, save_points, save_rois
    154         from ..process import setup_menus
    155         logger.debug("Started 'creating app.application.FlikaApplication.app'")

~/anaconda3/lib/python3.9/site-packages/flika/process/__init__.py in <module>
      3 logger.debug("Started 'reading process/__init__.py'")
      4 
----> 5 from .stacks import *
      6 from .math_ import *
      7 from .filters import *

~/anaconda3/lib/python3.9/site-packages/flika/process/stacks.py in <module>
      6 import numpy as np
      7 from qtpy import QtWidgets
----> 8 from ..window import Window
      9 from .. import global_vars as g
     10 from ..utils.BaseProcess import BaseProcess, BaseProcess_noPriorWindow, WindowSelector, MissingWindowError, CheckBox, SliderLabel, ComboBox

~/anaconda3/lib/python3.9/site-packages/flika/window.py in <module>
     11 from .utils.BaseProcess import WindowSelector, SliderLabel
     12 
---> 13 pg.setConfigOptions(useWeave=False)
     14 
     15 

~/anaconda3/lib/python3.9/site-packages/pyqtgraph/__init__.py in setConfigOptions(**opts)
     73     """
     74     for k,v in opts.items():
---> 75         setConfigOption(k, v)
     76 
     77 def getConfigOption(opt):

~/anaconda3/lib/python3.9/site-packages/pyqtgraph/__init__.py in setConfigOption(opt, value)
     62 def setConfigOption(opt, value):
     63     if opt not in CONFIG_OPTIONS:
---> 64         raise KeyError('Unknown configuration option "%s"' % opt)
     65     if opt == 'imageAxisOrder' and value not in ('row-major', 'col-major'):
     66         raise ValueError('imageAxisOrder must be either "row-major" or "col-major"')

KeyError: 'Unknown configuration option "useWeave"'
```

**Comment by kyleellefsen** [comment]
Thanks for the descriptions. I was able to recreate this error on a fresh install, and have fixed it in version 0.2.32. The cause was `pg.setConfigOptions(useWeave=False)` passed a configuration option that no longer works in the newer version of pyqtgraph. I'll close this issue now, but feel free to open if it you run into more issues.

**Comment by WeiyuChen-AhrensLab** [comment]
Hi Kyle,

Thank you very much for helping me with the issue! Flika has been very helpful for my work!

Best,
Weiyu

On Apr 25, 2021, at 5:39 PM, Kyle Ellefsen ***@***.******@***.***>> wrote:

External Email: Use Caution




Thanks for the descriptions. I was able to recreate this error on a fresh install, and have fixed it in version 0.2.32. The cause was pg.setConfigOptions(useWeave=False) passed a configuration option that no longer works in the newer version of pyqtgraph. I'll close this issue now, but feel free to open if it you run into more issues.

**Comment by csamuel11** [comment]
Thank you for fixing this issue!
